js/index.js

```diff
@@ -13,3 +13,38 @@ for (let i=0; i<skills.length; i++) {
   skill.innerHTML=skills[i];
   skillsList.appendChild(skill);
 };
+const messageForm = document.getElementsByName('leave_message');
```

Comment:
This method of DOM selection totally works, as your solution shows. If you want something where you don't have to access it by an index throughout the rest of your code, you could instead use
`const messageForm = document.querySelector('[name="leave_message"]');`
Then you can just write `messageForm.blahblahblah` instead of `messageForm[0].blahblahblah`.